content/en/blog/_posts/2019-12-09-csi-volume-migration-beta.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: blog
+title: "Kubernetes 1.17 Feature: Kubernetes In-Tree to CSI Volume Migration Moves to Beta"
+date: 2019-12-10
```

Comment:
> These 2 blogs, from SIG Storage, should be published today (9th): pending approval from the authors.
The publish date 2019-12-10; is that right?

Reply:
I'll fix this. It's today. Thanks